4/m_recaptchav2.cpp

```diff
@@ -0,0 +1,412 @@
+/*
+ * InspIRCd -- Internet Relay Chat Daemon
+ *
+ *   Copyright (C) 2015-2025 reverse Chevronnet
+ *   mike.chevronnet@gmail.com
+ *
+ * This file is part of InspIRCd.  InspIRCd is free software; you can
+ * redistribute it and/or modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; version 2.
+ */
+
+/// $ModAuthor: reverse mike.chevronnet@gmail.com
+/// $ModConfig: <captchaconfig conninfo="dbname=example user=postgres password=secret hostaddr=127.0.0.1 port=5432" 
+/// mode="master/slave" masterserver="master.example.com" url="http://meme.com/verify/" whitelistchan="#help,#support">
+/// $ModDepends: core 4
+
+/// $CompilerFlags: find_compiler_flags("libpq")
```

Comment:
It might be better if you use the InspIRCd SQL API instead of directly linking against postgres as its designed to integrate into the mainloop and not block connections if your SQL server is slow.